tests/topotests/conftest.py

```diff
@@ -320,6 +373,8 @@ def module_check_memtest(request):
     if request.config.option.memleaks:
         if get_topogen() is not None:
             check_for_memleaks()
+    check_for_backtraces()
+    check_for_core_dumps()
```

Comment:
So this is only going to be invoked after all tests in the module (the .py file) have run. I believe your intention is to have this run after each test -- at least I think that makes sense. How about we try and create a per function autouse:

```
@pytest.fixture(autouse=True, scope="function")
def post_test_checks(request):
    yield
    check_for_backtraces()
    check_for_core_dumps()
```
We should verify that the tests properly fail by having something abort. I'd just like to make sure that the `pytest.fail` is still able to fail a test during this autouse hook.

Reply:
nah I want it after the .py is finished.  That was intentional.  Just checking for backtraces or core files 1 time per .py is sufficient imo

Reply:
Ok, I think knowing which test actually caused the backtrace/core is useful info though.

Reply:
Frankly it's generally readily apparent if there is a crash once you know about it.  The data in the logs will give you what you need

Reply:
so eng does the work not the computer. ok. :)

This is fine if it bugs me enough I'll recode it some other time.